quic/s2n-quic-sim/src/batch.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <script src="https://cdn.jsdelivr.net/npm/vega@5"></script>
+  <script src="https://cdn.jsdelivr.net/npm/vega-lite@4"></script>
+  <script src="https://cdn.jsdelivr.net/npm/vega-embed@6"></script>
```

Comment:
Can you add an integrity hash for these?

Reply:
I'd have to pin to a particular version if we did. With the way it is now, we get minor and patch updates.